File: 2350-find-closest-number-to-zero/2350-find-closest-number-to-zero.py
class Solution:
    def findClosestNumber(self, nums: List[int]) -> int:
        l=[]
        for i in range(len(nums)):
            l.append(abs(nums[i]))
        r=min(l)
        close=[]
        for i in nums:
            if(abs(i)==r):
                close.append(i)
        return max(close)

File: 2350-find-closest-number-to-zero/2350-find-closest-number-to-zero.cpp
class Solution {
public:
    int findClosestNumber(vector<int>& nums) {
        vector<int>l;
        for (int i=0;i<nums.size();i++){
            l.push_back(abs(nums[i]));
        }
        int r=*min_element(l.begin(),l.end());
        vector<int>close;
        for (int i:nums){
            if(abs(i)==r){
                close.push_back(i);
            }
        }
        return *max_element(close.begin(),close.end());
        

        
    }
};